controllers/core/featureflagsource/controller.go

```diff
@@ -44,6 +44,9 @@ type FeatureFlagSourceReconciler struct {
 	FlagdProxy *flagdproxy.FlagdProxyHandler
 }
 
+// renovate: datasource=github-tags depName=open-feature/flagd/flagd-proxy
+const flagdProxyTag = "v0.3.0"
```

Comment:
i guess this can already be set to `v0.7.0` here

Reply:
just checked, and the latest proxy release seems to be 0.3.0, so please disregard my comment

Reply:
This is flagd-proxy, its version is just 0.3.0, see here https://github.com/open-feature/flagd/tags